README.md

```diff
@@ -69,6 +69,28 @@ wallet and sign its transactions.
 
 ## Structure
 
+```mermaid
+  graph TD;
+      cli-->app;
+      cli-->rpc_model;
+
+      app-->common;
+      app-->wollet;
+      app-->signer;
+      app-->tiny_jrpc;
+      app-->rpc_model;
+
+      jade-->common;
+
+      signer-->common;
+      signer-->jade;
+
+      wollet-->common;
+      
+      bindings-->signer;
+      bindings-->wollet;
+```
+
 Instead of going for a monolithic approach, we opted to split
 the projects in different components that might be useful
 independently:
```

Comment:
```suggestion
## Workspace Structure

The projects are split into different component crates that might be useful independently. 

Internal crate dependencies are shown in this diagram, where an arrow indicates "depends on": 

```mermaid
  graph TD;
      cli-->app;
      cli-->rpc_model;

      app-->common;
      app-->wollet;
      app-->signer;
      app-->tiny_jrpc;
      app-->rpc_model;

      jade-->common;

      signer-->common;
      signer-->jade;

      wollet-->common;
      
      bindings-->signer;
      bindings-->wollet;
```
## Crates 
```

Reply:
it seems code blocks don't do that well in suggest blocks